content/en/actions/app_builder/_index.md

```diff
@@ -50,21 +38,27 @@ Below are a few examples of what App Builder apps can do:
 - Use a PagerDuty integration to see who is on-call for each team in an organization.
 - Summarize the progress of each PR in a given repo.
 
-{{< img src="/service_management/app_builder/app-builder-blueprints-2.png" alt="App blueprints" style="width:100%;" >}}
+{{< img src="/service_management/app_builder/app-builder-app.png" alt="A screenshot of an app in App Builder" caption="An app in App Builder" style="width:100%;" >}}
+
+## Configure App Builder actions
+
+Datadog App Builder provides an [Action Catalog][1] of hundreds of actions across multiple integrations. The Action Catalog and the connection credentials for each integration are shared with [Datadog Workflow Automation][2]. If there isn't an integration that accomplishes your task, you can use generic actions such as the HTTP requests and JavaScript functions to perform any task that your app requires.
+
+{{< img src="/service_management/app_builder/app-builder-action-catalog-2.png" alt="A screenshot of App Builder after clicking Add Action" caption="The Action Catalog modal inside App Builder" style="width:100%;" >}}
```

Comment:
```suggestion
{{< img src="/service_management/app_builder/app-builder-action-catalog-2.png" alt="The Action Catalog modal inside App Builder" caption="The Action Catalog modal inside App Builder" style="width:100%;" >}}
```

Reply:
For this screenshot, you could consider:
- Whether it's necessary? I can see that it shows a bunch of the options you can choose when you're creating an app, but when a screenshot shows a user mid-procedure when it doesn't really talk about that step (that is, it's more illustrative than instructive), I sometimes wonder whether the image's existence is worth the maintenance.
- Whether you can trim the screenshot down to just the modal. If you/your PMs decide that the screenshot is necessary, that's cool and fine! But I might suggest cropping out the irrelevant parts of the screen. That way, if there's, say, a big reorg on the left nav, you don't have to worry about updating this screenshot so it's not out of date.